packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -319,6 +320,156 @@ class _BottomSheetState extends State<BottomSheet> {
 
 // See scaffold.dart
 
+typedef onChangeCallback<T> = void Function(T newValue);
+
+class _BottomSheetLayoutWrapper extends SingleChildRenderObjectWidget {
+  /// Creates a custom single child layout.
+  ///
+  /// The [delegate] argument must not be null.
+  const _BottomSheetLayoutWrapper({
+    super.key,
+    required this.animationValue,
+    required this.isScrollControlled,
+    this.onChildSizeChanged,
```

Comment:
required?

Reply:
They were required positional argument for the constructor of _ModalBottomSheetLayout (SingleChildLayoutDelegate), thus also making them required here.

Reply:
I mean should this.onChildSizeChanged also be required?

Reply:
Ah I see. Thanks! Done.